src/cmd.rs

```diff
@@ -968,6 +977,14 @@ impl Default for Commands {
                 p.then(optional(color()))
                     .map(|(_, rgba)| Command::SelectionFill(rgba))
             })
+            .command("selection/flip", "Flip selection", |p| {
+                p.then(word().label("h[orizontal]/v[ertical]"))
+                    .try_map(|(_, t)| match t.as_str() {
+                        "horizontal" | "h" => Ok(Command::SelectionFlip(Flip::Horizontal)),
+                        "vertical" | "v" => Ok(Command::SelectionFlip(Flip::Vertical)),
```

Comment:
I would prefer if this took `x` or `y` instead of `h` and `v`. For brush symmetry mode, we use `xsym` and `ysym`, so it would be more consistent.

Reply:
Oh, that actually makes more sense, I'll change that!

Reply:
alright, this should be resolved